constants/inter-latin.ts

```diff
@@ -0,0 +1,5 @@
+import { Inter } from '@next/font/google';
+
+export const inter = Inter({
+  subsets: ['latin'],
```

Comment:
As far as I see ([styleguide](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=2-127&t=yEX9oQA52QxcVppU-0)), we need to add `weight`:
- 400
- 500
- 700

Reply:
there is some problem, when I add weight for this font, and it's not required. I left without weight